### PR TITLE
Add faction epilogues 

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Content "Add new faction epilogues"

#### Purpose of change
The epilogue system is an underutilized part of the game, so I've decided to add new ones for several factions. 

#### Describe the solution
Add epilogues for the Norse Evangelicals, Isherwoods, Rabbit Enthusiasts, the Prisoners, and the Exodii.

#### Describe alternatives you've considered
Not adding any epilogues.

#### Testing
I launched the game, talked to a few people, and died. Everything works.

#### Additional context
None

### CODE REVIEW

**Comment by MNG-cataclysm** [comment]
Mi kno nnot houw tu spel. Eroors fexed.

File: data/json/snippets/epilogue_factions.json [hunk 1]
```diff
@@ -66,6 +66,46 @@
...
+        "id": "epilogue_faction_isherwoods_150",
+        "text": "    The generational land of the Isherwoods came to flourish following <the_cataclysm>, helping to feed nearby settlements.  Survivors flocked to learn botany, animal handling, and backwoods remedies, eventually forming a small community…"
+      },
+      {
+        "id": "epilogue_faction_lapin_0",
+        "text": "    Mr. Lapin continued his remote life in the wilderness, teaching whoever would visit.  The survivors who came slowly dwindled, until they stopped entirely.  Mr. Lapin was found dead several years later following a heart attack.  His body had decomposed to skeletal remains…"
+      },
+      {
+        "id": "epilogue_faction_lapin_150",
+        "text": "    The knowledge Mr. Lapin tought attracted a slew of individuals, who constructed makeshift housing to study with him.  As more people came, they formed a small community…"
```
Comment (Maleclypse) [inline data/json/snippets/epilogue_factions.json:92]:
```suggestion
        "text": "    The knowledge Mr. Lapin taught attracted a slew of individuals, who constructed makeshift housing to study with him.  As more people came, they formed a small community…"
```

**Comment by LyleSY** [comment]
Overall great stuff. This suggests that the Exodiii are here for the long haul, which is not my understanding of their whole deal. Can someone with more deep Exodii lore knowledge than me confirm this is good or suggest an improvement? Otherwise I see no issues

**Review by LyleSY — APPROVED** [review]

**Comment by MNG-cataclysm** [comment]
@LyleSY Could you link the discussion on the development Discord? I'd just like to read it.

**Pull request merged**